radiography/api/radiography.api

```diff
@@ -29,7 +29,7 @@ public final class radiography/ScanScopes {
 	public static final fun singleViewScope (Landroid/view/View;)Lradiography/ScanScope;
 }
 
-public abstract class radiography/ScannableView {
+public abstract interface class radiography/ScannableView {
```

Comment:
Also worth noting this break backward compatibility (as you can't switch over ScannableView anymore) but I don't expect any consumer to be impacted.

If we really really want to avoid that we could keep ScannableView as abstract class and make it implement an interface. But then I don't know how to name that interface

Reply:
I don't think it's possible to add direct subclasses to a sealed class in any way without technically breaking compatibility. Any consuming bytecode that assumes it has handled all the cases will be wrong.